sh/cmd.go

```diff
@@ -132,11 +132,14 @@ func run(env map[string]string, stdout, stderr io.Writer, cmd string, args ...st
 	c.Stderr = stderr
 	c.Stdout = stdout
 	c.Stdin = os.Stdin
-	log.Println("exec:", cmd, strings.Join(args, " "))
+
+	var quoted []string; for i:=0; i<len(args); i++ { 
+		quoted=append(quoted,fmt.Sprintf("\"%s\"",args[i]));
+	}
+	log.Println("[magefile-exec]:", cmd,  strings.Join(quoted, " "))
```

Comment:
for now, can you change this back to just "exec:" ?  I can understand the desire to change it to be more obvious, but I am hesitant to change any output that someone might depend on, without good reason.
```suggestion
	log.Println("exec:", cmd,  strings.Join(quoted, " "))
```